cell_based/src/simulation/numerical_methods/AbstractNumericalMethod.hpp

```diff
@@ -161,12 +161,12 @@ class AbstractNumericalMethod : public Identifiable
      * the simulation. The cell population and force collection pointers are then set by the simulation in
      * its constructor.
      */
-    AbstractNumericalMethod<ELEMENT_DIM,SPACE_DIM>();
+    AbstractNumericalMethod();
```

Comment:
I assume that this class used to be templated solely because its derived class(es) had templates

Reply:
The class is still templated:

```cpp
template<unsigned ELEMENT_DIM, unsigned SPACE_DIM=ELEMENT_DIM>
class AbstractNumericalMethod : public Identifiable
{ ... };
```

there were just unnecessary additional templates in the constructor and destructor. This was meaningless (but permitted_ until C++20, but GCC 14 started warning about the fact that it will be illegal from C++20.